lib/matplotlib/figure.py

```diff
@@ -454,10 +455,10 @@ def show(self, warn=True):
             except NonGuiException:
                 pass
         if warn:
-            import warnings
             warnings.warn(
-                "matplotlib is currently using a non-GUI backend, "
-                "so cannot show the figure")
+                ('matplotlib is currently using %s, which is a ' +
```

Comment:
Can you fix the docstring typo while you are working on this?

Reply:
sorry, I don't find the typo. Which line is it?

Reply:
"warm" should be "warn"...  Unless my screen is blurry.

Reply:
Yes... I think so. It is warn. :)

Reply:
Line 434/435

Reply:
oh I see it. Will fix. Thank you